static/js/views/discovered-addon.js

```diff
@@ -123,6 +129,55 @@ class DiscoveredAddon {
         controlDiv.innerHTML = el;
       });
   }
+  
+  /**
+   * Handle a click on the license button.
+   */
+  handleLicense(e) {
+    if(e.dataset.id) {
+      const license_url = "https://api.mozilla-iot.org:8443/addons/license/" + e.dataset.id;
```

Comment:
The base URL needs to be retrieved via `API.getAddonsInfo()`.

Reply:
Do you have an example of how that works?

Reply:
https://github.com/mozilla-iot/gateway/blob/master/static/js/views/settings.js#L1508

Reply:
By base url I take it you mean `https://api.mozilla-iot.org:8443/addons/license/`?

Reply:
`https://api.mozilla-iot.org:8443/addons` <-- that part